### PR TITLE
fix: thicker bottom border in labels list view

### PR DESCRIPTION
**Pull request by raastinn**

### CODE REVIEW

File: src/js/labels/components/Item.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { RemoveLabel } from "./Remove";
 const LabelItemBox = styled(BoxGroupSection)`
     align-items: center;
     display: flex;
+    bottom: -1px;
```
Comment (ReeceHoffmann) [inline src/js/labels/components/Item.tsx]: Bottom border visually looks good, but the top now has an extra px of space. See if you can find the root cause of the thicker border. (double border, thicker border css, ect) Fixing that will give you a more durable change in the case that other styling changes

Comment (raastinn) [reply src/js/labels/components/Item.tsx]: Yes, it is a double border. I'm trying to figure out what is causing it and how to get rid of it

Comment (igboyes) [reply src/js/labels/components/Item.tsx]: I'd suggest looking at other places the base components are used and see how this glitch is avoided.

If I am understand the situation properly, one possible solution is using `:last` to set the bottom border to `none` on the last child.

**Review by igboyes — APPROVED** [review]

**Pull request merged**